package exception1;

public class Exception04P348 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		//error case 4
		//out of bound. index is greater than array size
		int[] arr=new int[5];
		System.out.println(arr[5]);

	}

}
